src/views/partials/main/header.hbs

```diff
@@ -1,38 +1,41 @@
 <header>
-  <nav class="navbar fixed-top navbar-expand-lg navbar-dark bg-dark">
-    <a class="navbar-brand" href="/"><img class="navbar-logo" src="/img/navbar_logo.png"></a>
-    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarColor03" aria-controls="navbarColor03"
-      aria-expanded="false" aria-label="Toggle navigation">
-      <span class="navbar-toggler-icon"></span>
-    </button>
-    <div class="collapse navbar-collapse" id="navbarColor03">
-      <ul class="navbar-nav mr-auto">
-        <li class="nav-item">
-          <a class="nav-link" href="/learn">Lernen</a>
-        </li>
-        <li class="nav-item">
-          <a class="nav-link" href="/courses">Kurse</a>
-        </li>
-        <li class="nav-item">
-          <a class="nav-link" href="/workshops">Workshops</a>
-        </li>
-        <li class="nav-item">
-          <a class="nav-link" href="/about">About</a>
-        </li>
-      </ul>
-      <form class="form-inline my-2 my-lg-0" action="https://duckduckgo.com/">
-        <div class="form-group">
-          <div class="input-group">
-            <input type="search" class="form-control mr-sm-2" placeholder="Suche" name="q">
-            <input type="hidden" value="https://lernwerkstatt-site.azurewebsites.net/" name="sites">
-            <input type="hidden" value="1" name="kh">
-            <input type="hidden" value="1" name="kn">
-            <input type="hidden" value="1" name="kac">
-            <span class="input-group-btn"><button class="btn btn-secondary my-2 my-sm-0" type="submit">
-                <i class="fa fa-search" aria-hidden="true"></i></button>
-            </span>
+  <nav class="navbar fixed-top navbar-expand-lg navbar-light bg-primary">
+    <div class="container">
+      <a class="navbar-brand" href="/"><img height="24" src="/img/navbar_logo.png">Die Lernwerkstatt</a>
```

Comment:
Added `Die Lerwerkstatt` on first header link for SEO purposes.

Reply:
Can we hide it? It looked so clean without it :)